Tes avi

fahrizalrahman opened the pull request:
Review Code 

Pembenahan Laporan Detail & cetak & Download

- Total Harga di hapus 
- untuk data satuan konversi dipindah di kolom satuan 
- untuk harga subtotal dan lain lain dipindah align right

armun-sbm committed:
laporanhutangberedar

tampilan table yang angka adalah rata kanan

armun-sbm committed:
downloadlappembelianhutang

merubah judul dari kapital menjadi huruf kecil

haidarafif0809 committed:
Refactor Code Master Pelanggan & Penambahan Flafon Usia

- efisiensi query otoritas 
- penambahan kolom flafon usia

haidarafif0809 committed:
Merged branch tes_avi into tes_avi

haidarafif0809 committed:
Perbaikan Form Penjualan 

- perbaikan hitungan NaN
- seluruh angka di jadikan rata kanan
- menampilkan sisa plafon piutang 
- perbaikan perhitungan sisa plafon piutang saat di klik piutang.
-

haidarafif0809 committed:
Perbaikan Form Penjualan

- file form penjualannya lupa di add

haidarafif0809 committed:
Default Pajak Di Penjualan

fahrizalrahman committed:
Pembenahan Laporan Detail & cetak & Download   

- Total Harga di hapus 
- untuk data satuan konversi dipindah di kolom satuan 
- untuk harga subtotal dan lain lain dipindah align right

fahrizalrahman committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

haidarafif0809 committed:
Laporan Kekuatan Jual Peritem

haidarafif0809 committed:
Commit File Cache Class

haidarafif0809 committed:
Merged branch tes_avi into tes_avi

RindangRamadhan committed:
Kartu Stok dan Cetak Penjualan

- Menambahkan separator dan rata kanan untuk data berupa angka

RindangRamadhan committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

hendriandaglos commented:
Mas Rizal (Pembenahan Laporan Detail & cetak & Download)
1. Download lap penjualan detail jangan di kasih pemisah titik
2. laporan_penjualan_piutang.php
	- rata kanan belom di kasih
	- ketika di klik rekap / rekap semua table nya kosong 
	- cetak nya belum rata kanan
	- download jangan di kasih tanda pemisah titik

fahrizalrahman committed:
Perbaikan Review Code (Hendri)

- 1. Download lap penjualan detail jangan di kasih pemisah titik
2. laporan_penjualan_piutang.php
    * rata kanan belom di kasih
    * ketika di klik rekap / rekap semua table nya kosong
    * cetak nya belum rata kanan
    * download jangan di kasih tanda pemisah titik

fahrizalrahman committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

haidarafif0809 committed:
Perbaikan Laporan Laba Kotor Penjualan

haidarafif0809 committed:
Merged branch tes_avi into tes_avi

haidarafif0809 committed:
Perhitungan Diskon Bertingkat Form Penjualan dan Proses Bayar nya 

- penambahan perhitungan diskon bertingkat , ketika menambah , menghapus dan mengedit tbs nya .
- Penambahan kolom potongan persen di table penjualan .

haidarafif0809 committed:
Ubah Inner Join Pelanggan Jadi Left Join (laporan penjualan rekap)

haidarafif0809 committed:
Cetak Penjualan Besar Penambahan Diskon Bertingkat

haidarafif0809 committed:
Pembulatan Diskon Bertingkat Penjualan

haidarafif0809 committed:
Perbaikan Perhitungan Diskon Nominal Untuk Diskon bertingkat
